packages/myst-cli/src/process/mdast.ts

```diff
@@ -85,13 +85,6 @@ const htmlHandlers = {
   },
 };
 
-export type PageReferenceStates = {
-  state: ReferenceState;
-  file: string;
```

Comment:
All of this stuff (file, url, etc) is now included directly on the `ReferenceState`, so we don't need another "state" class that includes "state" 🧹